src/otaclient/configs/_cfg_consts.py

```diff
@@ -54,6 +54,8 @@ def ACTIVE_ROOT(self) -> str:  # NOSONAR
     # this path should not be included in the mounted rootfs
     DYNAMIC_CLIENT_SQUASHFS_FILE = "/.otaclient.squashfs"
 
+    OTAPROXY_CACHE_DIR = "/ota-cache"
```

Comment:
Let's re-use this const at the following code 👍 
https://github.com/tier4/ota-client/blob/9a02409b256cb0fc6846a91c478eb69367155642/src/ota_proxy/config.py#L25